Resources/Prototypes/Entities/Structures/Walls/girders.yml

```diff
@@ -6,7 +6,7 @@
   components:
   - type: Transform
     anchored: true
-    noRot: true
+    noRot: false
```

Comment:
are you certain this will not cause issues for other walls?

Reply:
When I tested I my end there wasn't any funny issues I could find. Magil said he would test it on his end too. Just waiting to see what he says.